res/lang/programme/programme_en.txt

```diff
@@ -106,46 +106,46 @@ PROGRAMME_PRODUCT_SHOW        = Show
 
 OPENINGSHOW_TITLE       = Big launch of "%CHANNELNAME%"
-OPENINGSHOW_DESCRIPTION = Let's pop the corks. %CHANNELNAME% is launching today.
+OPENINGSHOW_DESCRIPTION = Let's pop the corks! %CHANNELNAME% is launching today.
 
 SPORT_PROGRAMME_TITLE                                = %SPORTNAME%: %LEAGUENAME% %SEASONYEARSTART%
 SPORT_PROGRAMME_ALL_X_MATCHES_OF_LEAGUEX_IN_SEASON_X = All %MATCHCOUNT% matches of %LEAGUENAME% in season %SEASONYEARSTART%.
 SPORT_PROGRAMME_MATCH_TIMES                          = Match times
-SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between place %TEAM1RANK% and %TEAM2RANK% of the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
-SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star %TEAM1STARSHORT% against the team around top scorer %TEAM2STAR%.
+SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between teams placed %TEAM1RANK% and %TEAM2RANK% in the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
+SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star, %TEAM1STARSHORT%, against top scorer %TEAM2STAR%'s team.
 SPORT_PROGRAMME_MATCH_DESCRIPTION3                   = An interesting game is coming up when %TEAM1ARTICLE1% %TEAM1NAME% meet the players of %TEAM2NAMEINITIALS%.
 SPORT_PROGRAMME_MATCH_DESCRIPTION4                   = %TEAM2ARTICLE1% %TEAM2NAME% visiting %TEAM1NAMEINITIALS% in %TEAM1CITY%.
-SPORT_PROGRAMME_MATCH_DESCRIPTION5                   = %TEAM2TRAINER% and his team travel to %TEAM2CITY%, rank %TEAM2RANK% in the league.
+SPORT_PROGRAMME_MATCH_DESCRIPTION5                   = %TEAM2TRAINER% and his team travel to %TEAM2CITY%, placed %TEAM2RANK% in the league.
 
-SPORT_PROGRAMME_PLAYOFF_MATCH    = Playoff match.
-SPORT_PROGRAMME_MATCH_OF_LEAGUEX = Match of %LEAGUENAME%.
-SPORT_PLAYOFFS_LONG              = "Playoffs"
-SPORT_PLAYOFFS_SHORT             = "PO"
+SPORT_PROGRAMME_PLAYOFF_MATCH    = Relegation match.
+SPORT_PROGRAMME_MATCH_OF_LEAGUEX = Match taking place in %LEAGUENAME%.
+SPORT_PLAYOFFS_LONG              = "Relegation"
+SPORT_PLAYOFFS_SHORT             = "Rel"
 
-SPORT_AFTER_PENALTY_SHORT  = "PSO"
+SPORT_AFTER_PENALTY_SHORT  = "Penalties"
 SPORT_AFTER_OVERTIME_SHORT = "AET"
 
-LIVE_ON_TAPE                        = Live on tape
+LIVE_ON_TAPE                        = Live recording
 ALL_MATCHES_FINISHED                = All matches finished
-X_OF_Y_LIVE_MATCHES_ALREADY_STARTED = %X%/%Y% Live matches already started.
+X_OF_Y_LIVE_MATCHES_ALREADY_STARTED = %X%/%Y% live matches already started.
 
 
 PENALTY_OF_X_WAS_PAID                                      = Penalty of %MONEY% was paid.
-BROADCAST_OF_XRATED_PROGRAMME_X_NOT_ALLOWED_DURING_DAYTIME = Broadcast of x-rated programme |b|%TITLE%|/b| not allowed during daytime.
+BROADCAST_OF_XRATED_PROGRAMME_X_NOT_ALLOWED_DURING_DAYTIME = Broadcast of X-rated programme |b|%TITLE%|/b| not allowed during daytime.
```

Comment:
Above you corrected t o"X-Rated", shouldn't it then either be "X-Rated" here too ? I understand that "X-rated" is not a choice too.

Reply:
Yeah, I'll change it.